torch/_functorch/eager_transforms.py

```diff
@@ -354,6 +356,8 @@ def jacrev(func: Callable, argnums: Union[int, Tuple[int]] = 0, *, has_aux=False
             the function to be differentiated and the second element is
             auxiliary objects that will not be differentiated.
             Default: False.
+        chunk_size (None or int): If specified, controls the maximum size of chunk for computing
+            Jacobian. Default: None.
```

Comment:
Need to tweak it.

Specify what happens for `None`.

Reply:
If None (default), we will use the maximum chunk size (this is equivalent to doing a single vmap over vjp to compute the jacobian). If not None, then we will compute the jacobian chunk_size rows at a time using vmap to vectorize the computation. Note that chunk_size=1 is equivalent to computing the jacobian row-by-row with a for-loop. If you run into memory issues computing the jacobian, please try to specify a non-None chunk_size.

Something like that